src/ui/user/UserDetailsModal.js

```diff
@@ -161,6 +179,13 @@ export default class UserDetailsModal extends Component {
                         player={user.name}
                         onSelect={this._onSelectGame}/>
                     </div> : null}
+                  {tab === 'rankGraph' ?
+                    <div className='UserDetailsModal-rank-graph'>
+                      {this.state.graph ?
```

Comment:
As mentioned above, a separate component means you can just use <RankGraph ... /> and let that component figure out the lazy loading